contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -115,6 +119,15 @@ def queue_get_return_none_when_empty():
                 create_associated_file_objects(kolibrinode, node)
 
 
+def fuzz(title_description):
+    """
+    Apply porter stemming algorithm then double metaphone algorithm to the passed in String
+    to obtain normalized and misspelling tolerant hash values/tokens
+    """
+    processed_tokens = [doublemetaphone(self.stemmer.stem(word)) for word in title_description.split()]
```

Comment:
So that this "fuzzing" logic isn't defined in two different places, it would be nice to put it in something like `content/utils/search.py` and use it from here as well as within Kolibri.